homeassistant/components/zha/switch.py

```diff
@@ -72,7 +71,7 @@ def __init__(
     ) -> None:
         """Initialize the ZHA switch."""
         super().__init__(unique_id, zha_device, channels, **kwargs)
-        self._on_off_channel: OnOffChannel = self.cluster_channels.get(CHANNEL_ON_OFF)
+        self._on_off_channel = self.cluster_channels[CHANNEL_ON_OFF]
```

Comment:
The type hint here actually throws mypy off.
My understanding here is also that CHANNEL_ON_OFF is always present for switches.

Reply:
Yes, it should be there, otherwise it would not match the entity.